pkg/spiffs/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME=spiffs
+PKG_URL=https://github.com/pellepl/spiffs.git
+PKG_VERSION=39937743fbbec4b82308ee08332bf9180408d23b
```

Comment:
sorry, I did not see the tag on the Github mobile page. I agree this is the commit we should use.